<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizWindow</class>
 <widget class="QMainWindow" name="QuizWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quiz AI - Spørgsmål</string>
  </property>
  <property name="styleSheet">
   <string>QMainWindow {
    background-color: #f0f0f0;
}

QLabel {
    font-size: 12pt;
    color: #333;
    padding: 10px;
}

QPushButton {
    font-size: 11pt;
    padding: 12px;
    margin: 5px;
    border: 2px solid #ddd;
    border-radius: 8px;
    background-color: white;
    text-align: left;
}

QPushButton:hover {
    background-color: #e3f2fd;
    border-color: #2196f3;
}

QPushButton:checked {
    background-color: #bbdefb;
    border-color: #1976d2;
}

QPushButton#next_btn {
    background-color: #4CAF50;
    color: white;
    border: none;
    text-align: center;
}

QPushButton#next_btn:hover {
    background-color: #45a049;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="topic_label">
      <property name="text">
       <string>Emne: </string>
      </property>
      <property name="styleSheet">
       <string>font-size: 14pt; font-weight: bold; color: #2E7D32;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="question_label">
      <property name="text">
       <string>Spørgsmål indlæses...</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string>font-size: 14pt; font-weight: bold; background-color: white; border: 1px solid #ddd; border-radius: 5px;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="option_a_btn">
      <property name="text">
       <string>A) </string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="option_b_btn">
      <property name="text">
       <string>B) </string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="option_c_btn">
      <property name="text">
       <string>C) </string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="option_d_btn">
      <property name="text">
       <string>D) </string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="result_label">
      <property name="text">
       <string></string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string>font-size: 12pt; background-color: white; border: 1px solid #ddd; border-radius: 5px;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="submit_btn">
        <property name="text">
         <string>Svar</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_btn">
        <property name="text">
         <string>Næste Spørgsmål</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
